content/tutorials/manage-the-rippled-server/installation/capacity-planning.md

```diff
@@ -13,40 +13,48 @@ To meet your `rippled` capacity requirements, you must address these technical f
 - The [configuration settings](#configuration-settings) that affect resource utilization
 - The [network and hardware](#network-and-hardware) requirements to achieve consistent, good performance across the XRP Ledger network
 
+
```

Comment:
I know it's outside of the scope for this PR, but the intro could probably be reworded to more closely align with the rest of the content, especially the very first paragraph.